getting-started/02.ACTIONS.md

```diff
@@ -0,0 +1,41 @@
+# Actions
+
+1. [Prompts](./00.PROMPTS.md)
+2. [Templates](./01.TEMPLATES.md)
+3. [**Actions**](./02.ACTIONS.md)
+4. [Plans](./03.PLANS.md)
+5. [Chaining](./04.CHAINING.md)
+
+A function that can be called by a plan or template.
+
+## Template
+
+```Prompt
+human - remind me to buy milk
+ai - DO addItem list="groceries" item="milk" SAY Ok I added milk to your groceries list
+```
```

Comment:
The prompt example is `addItem` action, but the code example below is `LightsOn` action. Better to use the same action?